packages/bull-vault/src/BullStrategy.sol

```diff
@@ -195,6 +200,10 @@ contract BullStrategy is ERC20, LeverageBull {
         emit RedeemCrabAndWithdrawEth();
```

Comment:
do we need anything in event, prb should add the crab and wpowerperp and eth redeemed?

Reply:
added